pom.xml

```diff
@@ -266,6 +266,28 @@ limitations under the License.
           <ignoreNonCompile>true</ignoreNonCompile>
         </configuration>
       </plugin>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-enforcer-plugin</artifactId>
```

Comment:
is this overriding the configuration in the parent pom?

Reply:
The checks in parent pom should still be propagated unless we specifically disable it according to https://maven.apache.org/guides/mini/guide-configuring-plugins.html#using-the-inherited-tag-in-build-plugins

I also tested by  removing https://github.com/googleapis/java-bigtable-hbase/blob/bigtable-1.x/bigtable-dataflow-parent/bigtable-beam-import/pom.xml#L259 and got the dependency errors

Reply:
excellent, thank you :)